webpages/settings/licenses.html

```diff
@@ -23,6 +23,7 @@ <h1 v-cloak>{{ msg("licenses") }}</h1>
     </div>
     <div class="main">
       <div class="licenses-block">
+        <h2 id="loading">Loading...</h2>
```

Comment:
I don't think this is necessary. On a device that takes at least 6 seconds to load the popup the licensees page still loads in less than one. It doesn't hurt to keep though.

Reply:
Do we want to localize this?

Reply:
On my laptop, the loading text only appears for a few frames, which is ugly.

Reply:
I just changed it:
- "Loading..." is localized
- "Loading..." is not displayed for 250ms